pom.xml

```diff
@@ -92,6 +92,7 @@
 
     <modules>
         <module>api</module>
+        <module>bundle</module>
```

Comment:
Should the bundle module always be active?

Reply:
I figured yes for the build and unit tests, but the actual bundle generation will only happen when the `generate-bundle` is active.